assets/js/app.js

```diff
@@ -0,0 +1,19 @@
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
```

Comment:
Please use ES6 syntax for instance instead of using `var` use `Let `or `const` where necessary.